io/common/include/detray/io/file_handle.hpp

```diff
@@ -0,0 +1,66 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// System include(s)
+#include <cassert>
+#include <cstdint>
+#include <fstream>
+#include <ios>
+#include <iostream>
+#include <limits>
+#include <string>
+
+namespace detray::io::detail {
+
+/// Wrapper around a file stream
+class file_handle final {
```

Comment:
Would it make sense to put this file in the `io/common/include/detray/io/detail/` subdirectory instead?

Reply:
Certainly